src/KafkaFlow.Abstractions/IEvent.cs

```diff
@@ -9,9 +9,9 @@
     public interface IEvent
     {
         /// <summary>
-        /// Subscribes to the event.
+        /// Subscribes to the async event.
         /// </summary>
-        /// <param name="handler">The handler to be called when the event is fired.</param>
+        /// <param name="handler">The handler to be called when the async event is fired.</param>
         /// <returns>Event subscription reference</returns>
```

Comment:
Maybe we could leave the summary as before.

Reply:
Done, please review